src/stores/index.js

```diff
@@ -139,9 +139,14 @@ class RootStore {
       metadata.public = metadata.public || {};
       metadata.public.asset_metadata = metadata.public.asset_metadata || {};
 
+      metadata.searchables = (
```

Comment:
Instead of making a separate call, you can add `searchables` to the select list of the ContentObjectMetadata call above.

The `select` attribute specifies 'only return metadata at these keys'. The existing metadata call is retrieving, in one request, public/asset_metadata and any other attributes in other areas of metadata. Adding 'searchables' to the select list will have the same effect.